plugins/CoreHome/javascripts/dataTable.js

```diff
@@ -107,7 +109,7 @@ $.extend(DataTable.prototype, UIControl.prototype, {
         this.workingDivId = this._createDivId();
         domElem.attr('id', this.workingDivId);
 
-        this.maxNumRowsToHandleEvents = 255;
+        this.maxNumRowsToHandleEvents = 255; // TODO: need to remove this, but need more data to test
```

Comment:
here's another todo but we might still need this or not? Or did you make it all faster?

Reply:
I didn't make it faster, but the comparison row will usually have more than this number of rows on some reports, so I'll have to look into making it faster. The TODO is there to remind me.